README.md

```diff
@@ -43,22 +55,34 @@ There are currently many excellent plotting libraries in Python. The main ones a
 * [HoloViews](http://ioam.github.io/holoviews/)
 * [VisPy](http://vispy.org/)
 
-Each of these libraries does a certain set of things really well.
-However, such a proliferation of options creates great difficulty for users as they have to wade through all of these APIs to find which of them is the best for the task at hand.
-For individuals just learning data science, this forces them to focus on learning APIs rather than exploring data.
-
-Another challenge is that all of the current APIs require the user to write code, even for the simplest of visualizations.
-This is unfortunate and unnecessary as the type of visualization (histogram, scatterplot, etc.) can often be inferred with basic information such as the columns of interest and the data types of those columns.
-For example, if you are interested in a visualization of two numerical columns, a scatterplot is almost certainly a good starting point.
-If you add a categorical column to that, you probably want facetted scatterplots.
-In cases where the visualization can't be inferred, simple user interfaces can enable the construction of visualizations without any coding. [Tableau](http://www.tableau.com/) and the [Interactive Data Lab's](http://idl.cs.washington.edu/)
-[Polestar](https://github.com/vega/polestar) and [Voyager](https://github.com/vega/voyager) are
-excellent examples of such UIs.
-
-We feel that these challenges can be addressed without creation of yet another plotting library.
-The approach of Altair is to build visualizations using a layered approach that leverages the full capabilities of existing visualization libraries:
-
-1. A constrained and simple Python API (Altair) that is purely declarative and emits JSON that follows the vega-lite spec.
+Each of these libraries does a certain set of things really well. However, such a
+proliferation of options creates great difficulty for users as they have to wade
+through all of these APIs to find which of them is the best for the task at hand.
+Because none of these libraries are optimized for high-level statistical
+visualization, users have to assemble their own using a mish-mash of APIs. For
+individuals just learning data science, this forces them to focus on learning APIs
+rather than exploring data.
+
+Another challenge is that all of the current APIs require the user to write code,
```

Comment:
Comments addressed...

On Sat, Jun 4, 2016 at 9:09 PM, Jake Vanderplas <notifications@github.com>
wrote:

> In README.md
> <https://github.com/ellisonbg/altair/pull/122#discussion_r65811050>:
>
> > -[Polestar](https://github.com/vega/polestar) and [Voyager](https://github.com/vega/voyager) are
> > -excellent examples of such UIs.
> > -
> > -We feel that these challenges can be addressed without creation of yet another plotting library.
> > -The approach of Altair is to build visualizations using a layered approach that leverages the full capabilities of existing visualization libraries:
> > -
> > -1. A constrained and simple Python API (Altair) that is purely declarative and emits JSON that follows the vega-lite spec.
> > +Each of these libraries does a certain set of things really well. However, such a
> > +proliferation of options creates great difficulty for users as they have to wade
> > +through all of these APIs to find which of them is the best for the task at hand.
> > +Because none of these libraries are optimized for high-level statistical
> > +visualization, users have to assemble their own using a mish-mash of APIs. For
> > +individuals just learning data science, this forces them to focus on learning APIs
> > +rather than exploring data.
> > +
> > +Another challenge is that all of the current APIs require the user to write code,
>
> Maybe we should clarify here: the challenge is not the writing of code *per
> se*, but that the user often must explicitly specify many incidental
> aspects of the visualization.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ellisonbg/altair/pull/122/files/188a6abd2b97eeba574827651528888568c43eb1#r65811050>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABr0J7O6e3YChc57hDWtrabhcgVig6Pks5qIkwRgaJpZM4IuLRd>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com